README.md

```diff
@@ -42,7 +42,7 @@ When updating this, also update:
 
 Alloy will keep a rolling MSRV (minimum supported rust version) policy of **at
 least** 6 months. When increasing the MSRV, the new Rust version must have been
-released at least six months ago. The current MSRV is 1.65.0.
+released at least six months ago. The current MSRV is 1.68.
```

Comment:
aws-sdk-kms 1.68, with msrv at [1.70 in readme](https://github.com/awslabs/aws-sdk-rust#supported-rust-versions-msrv). I think we'll want to bump this to latest stable for AFIT anyway